packages/components/package-lock.json

```diff
@@ -0,0 +1,4958 @@
+{
```

Comment:
as i understand we don't need lock file for components, right @peterhass ?
also we are using yarn for this project